lib/pure/terminal.nim

```diff
@@ -462,15 +509,35 @@ elif not defined(windows):
     result = stdin.readChar()
     discard fd.tcsetattr(TCSADRAIN, addr oldMode)
 
+# Deprecated procs assuming output to stdout:
+template setCursorPos*(x, y: int) {.deprecated.} = setCursorPos(stdout, x, y)
```

Comment:
@Araq Only versions with implicit `stdout` are deprecated, there are new versions of this procs taking 1st arg file handle. As I said in the commit comments there is rationale behind that - `stdout` is not always the file handle which is connected to the TTY, `stderr` is more likely that handle.

It is just to avoid stupid mistakes, eg.: was using `stderr` everywhere but have chosen `setCursorPos` with implicit `stdout` and something doesn't work.